pup-0002.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Docstring Style Conventions
+author: Austin Macdonald
+created: 28-Mar-2017
+status: proposed
+---
+
+## Summary
+
+This PUP proposes the use of Google style docstrings for Pulp 3+.
+
+## Motivation
+
+To improve docstring readability and increase information density.
```

Comment:
I think this deserves more explanation. In particular, since this would be a change from one style to another, this proposal should explain the pros and cons of making the switch as compared to staying with the current style.

Even if the discussion is limited to "The current style X would be replaced by style Y. Most current Pulp developers find style Y to be more readable than style X.", that would be very valuable to include.

Additionally, some mention of the current style in the "Alternatives" section would be appropriate.